ui/page.go

```diff
@@ -68,7 +68,6 @@ func (win *Window) addPages() {
 	win.pages[PageRestore] = win.RestorePage
 	win.pages[PageSend] = win.SendPage
 	win.pages[PageSignMessage] = win.SignMessagePage
-
 }
```

Comment:
you should avoid making changes to code that's barely related to what you're working on.